snapcraft/tests/test_yaml.py

```diff
@@ -734,6 +734,11 @@ def test_valid_app_daemons(self):
             'service3': {
                 'command': 'binary3',
                 'daemon': 'forking',
+            },
+            'service4': {
+                'command': 'binary4',
+                'daemon': 'simple',
+                'restart-condition': 'always',
```

Comment:
Not sure how important the watchdog test is... I guess it's possible to be added later. Maybe we don't want the brittleness there.